warehouse/accounts/models.py

```diff
@@ -96,6 +96,9 @@ class User(SitemapMixin, HasEvents, db.Model):
     totp_secret = Column(LargeBinary(length=20), nullable=True)
     last_totp_value = Column(String, nullable=True)
 
+    # XXX: Can be removed once OIDC is removed from beta.
+    has_oidc_beta_access = Column(Boolean, nullable=False, server_default=sql.false())
```

Comment:
Bleh, I'm -1 on a column directly on the User model, but won't block the PR on this.

The aspiration was a more extensible AdminFlags or similar to manage this kind of thing, but obviously that hasn't come to fruition yet.

Reply:
AdminFlags are just global flags that can be set by admins -- they don't allow us to set per-user flags (unless we want to overhaul them, which is probably out of scope here).

We tried to do this with ACLs but given how this currently works with the User model, it was a bit more janky than this, which will be temporary.